modules/util/client-extension-util-spring-boot3/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
 	implementation group: "com.liferay", name: "com.liferay.petra.string", version: "latest.release"
 	implementation group: "org.json", name: "json", version: "20231013"
 	testImplementation group: "com.liferay", name: "com.liferay.petra.function", version: "latest.release"
+	testImplementation group: "com.liferay", name: "com.liferay.petra.string", version: "latest.release"
```

Comment:
I think we need to remove petra-string from `implementation` here.

Reply:
Petra is still used in unit tests.